combined-javadoc/build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+    id 'io.deephaven.project.basic'
+    id 'java'
+}
+
+configurations {
+    combinedJavadoc
+}
+
+String javaDocOverviewLocation = 'build/docs/overview.html'
+
+def gitHash
+gitHash = "${-> gitHash = 'git rev-list --max-count=1 HEAD'.execute([], rootDir).text.trim()}"
```

Comment:
Just in case, might want to fall back on some default strings, such as if someone downloads a zip of the project from github rather than clone.

Reply:
Yeah, it's a good point - the good news is that it's not part of the prepareCompose dependencies, and typically only run in CI. I can create an issue for future.

Reply:
#2000